ts/lib/auth/googleauth.ts

```diff
@@ -400,9 +406,12 @@ export class GoogleAuth {
     // Set the JSON contents
     this.jsonContent = json;
     if (json.type === 'authorized_user') {
-      client = new UserRefreshClient();
+      client = new UserRefreshClient(
+          undefined, undefined, undefined, this.refreshTokenEarlyMillis);
```

Comment:
Similar to the discussion about switching to an options object in the oAuth2Client constructor, we should probably make that change here too.  WDYT @ofrobots ?